user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/Objects/ApiDefinition.md

```diff
@@ -0,0 +1,103 @@
+
+|Property   |Type      |Explanation|
+|-----------|----------|-----------|
+|ScriptName |string    |Name of the script that should be executed. Cannot be null or empty.|
+|InputType  |InputType |Determines how the input data from the request will be handled. The options are 'Parameters' or 'RawBody'. See [Defining a new API](xref:UD_APIs_Define_New_API) for more info.|
+
+#### AutomationScriptNoEntryPoint
+
+This `ActionType` is the same as the previous one, with the difference that there is no need to define a special entry point method in your automation script. The script will be executed through the default Run() method. This means that the `ApiTriggerInput` object is not available, and it is also not possible to return the `ApiTriggerOutput` to the API user. It will return 200 OK with an empty response body in case the script succeeded, and an `AutomationActionError` response error when the script fails. An example of how to use this action can be found [here](xref:UD_APIs_Using_existing_scripts#using-the-script-without-the-onapitrigger-entry-point).
+
+This action will always convert the request body to a `Dictionary<string, string>`. It will verify that there is a value for each of the script parameters defined on the script, in the parsed dictionary. More info [here](xref:UD_APIs_Define_New_API#user-input-data).
+
+You'll need to assign an `AutomationScriptNoEntryPointActionMeta` instance to the `ActionMeta` property. This has the following fields:
+
+|Property   |Type      |Explanation|
+|-----------|----------|-----------|
+|ScriptName |string    |Name of the script that should be executed. Cannot be null or empty.|
+
+## Requirements
+
+- **Create & Update**
+  - The `Name` property must not contain only white space characters. It must either contain visible characters or be empty/null.
+  - The `Route` property must meet the requirements mentioned in the [Route](#route) section above.
+  - The `ActionMeta` property must contain an instance of a meta class according to the configured action type. See the [Actions](#actions) section above.
+
+## Errors
+
+When something goes wrong during the CRUD actions, the `TraceData` can contain one or more `ApiDefinitionErrors`. Below is a list of all possible `ErrorReasons`. The `Id` property of the `ApiDefinitionError` object will always contain the ID of the API definition that could not be created or updated.
+
+|Reason      |Description|
+|------------|-----------|
+|InvalidName |The given name only contained white space characters.|
+|RouteInUse |The given route is already used by another `ApiDefinition`. *ConflictingDefinitionId* contains the ID of the definition that is already using the route. *Route* contains the invalid route.|
```

Comment:
```suggestion
|RouteInUse |The given route is already used by another `ApiDefinition`. The *ConflictingDefinitionId* property contains the ID of the definition that is already using the route. The *Route* property contains the invalid route.|
```

Reply:
Same here, would prefer to keep this, a bit shorter to keep the table simple & consistent with the more recent DOM docs.